src/main/java/emcshop/gui/ProfileLoader.java

```diff
@@ -486,7 +486,7 @@ public PlayerProfile load(String playerName) throws IOException {
             //@formatter:off
             return new PlayerProfile.Builder()
                     .playerName(properties.get("name"))
-                    .private_(properties.getBoolean("private", false))
+                    .setPrivate(properties.getBoolean("private", false))
```

Comment:
That method is part of a "builder" class.  Builder class method names do not start with "set".  See this page for information on the builder design pattern.

http://javarevisited.blogspot.com/2012/06/builder-design-pattern-in-java-example.html